md/templates/md.html

```diff
@@ -47,24 +47,24 @@
       <th class="text-right">Stops</th>
     </tr>
     <tr>
-      <td><a href="{% url "md:agency-detail" "80" %}">MSP</a></td>
-      <td class="text-right">641,952</td>
+      <td><a href="{% url "md:agency-detail" "77" %}">Maryland State Police</a></td>
```

Comment:
Look at the first two changes:  80/MSP  and 77/Montgomery turned into 77/Maryland State Police and 80/Montgomery County Police Department.  MSP came after Montgomery alphabetically but Maryland comes before.  The table is ``ORDER BY agency_description``.